python/lsst/meas/extensions/convolved/convolved.py

```diff
@@ -75,8 +75,8 @@ def __init__(self, name, schema, seeing, config, metadata):
         kronKeys = Struct(
             result=lsst.meas.base.FluxResultKey.addFields(schema, name + "_kron",
                                                           doc="convolved Kron flux: seeing %f" % (seeing,)),
-            flag = schema.addField(name + "_kron_flag", type="Flag",
-                                   doc="convolved Kron flux failed: seeing %f" % (seeing,)),
+            flag=schema.addField(name + "_kron_flag", type="Flag",
+                                 doc="convolved Kron flux failed: seeing %f" % (seeing,)),
```

Comment:
Perhaps for next time, `schema.join(name, 'kron', 'flag')` would be the preferred way to concatenate schema name components these days.  We haven't made a push to use it across the board, though, so I won't insist on doing it on a high-priority fix ticket like this one.